app/models/outcome.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class Outcome
```

Comment:
I wonder if we should be explicit about the purpose of this class. `Outcome` is quite broad, `OutcomeDetail` or `OutcomeDate`?

Reply:
👍  I was initially thinking this would be responsible for the `outcome` itself as well (e.g. `passed`) but now i'm thinking not. I'll go with `OutcomeDate`!